### PR TITLE
Fix Inspector and WidgetRebuildIndicators for bazel projects.

### PR DESCRIPTION
**Pull request by jacob314**

### CODE REVIEW

File: src/io/flutter/inspector/EvalOnDartLibrary.java [hunk 1]
```diff
@@ -44,7 +44,7 @@
   CompletableFuture<LibraryRef> libraryRef;
   private final Alarm myRequestsScheduler;
 
-  static final int DEFAULT_REQUEST_TIMEOUT_SECONDS = 5;
+  static final int DEFAULT_REQUEST_TIMEOUT_SECONDS = 10;
```
Comment (jacob314) [inline src/io/flutter/inspector/EvalOnDartLibrary.java]: with the other fixes, the timeout doesn't really need to be raised but raising it anyway just to be safe.

File: src/io/flutter/bazel/Workspace.java [hunk 2]
```diff
@@ -33,6 +33,8 @@
 public class Workspace {
   private static final String PLUGIN_CONFIG_PATH = "dart/config/intellij-plugins/flutter.json";
 
+  public static final String BAZEL_URI_SCHEME = "google3://";
```
Comment (jacob314) [inline src/io/flutter/bazel/Workspace.java]: @jakemac53 please let me know if you clean this up and just make the scheme bazel://

File: src/io/flutter/sdk/FlutterSdk.java [hunk 3]
```diff
@@ -548,92 +548,4 @@ public void onTextAvailable(@NotNull ProcessEvent event, @NotNull Key outputType
 
     return null;
   }
-
-  /**
-   * A {@link FlutterSdk} that is compatible with the Bazel version of the Dart and Flutter SDK.
-   */
-  public static class BazelSdk extends FlutterSdk {
```
Comment (jacob314) [inline src/io/flutter/sdk/FlutterSdk.java]: This class wasn't ever used and doesn't seem compatible with current implementations.

File: testSrc/unit/io/flutter/perf/FlutterWidgetPerfTest.java [hunk 4]
```diff
@@ -332,10 +351,10 @@ public void testFileStatsCalculation() throws InterruptedException, ExecutionExc
         perfModels.put(textEditor.getName(), model);
         return model;
       },
-      path -> new FakeFileLocationMapper()
+      path -> new FakeFileLocationMapper(path)
     );
     widgetPerfProvider.simulateWidgetPerfEvent(PerfReportKind.rebuild,
-                                               "{\"startTime\":1000,\"events\":[1,1,2,1,3,1,4,1,6,1,10,4,11,4,12,4,13,1,14,1,95,1,96,1,97,6,100,6,102,6,104,6,105,1,106,1],\"newLocations\":{\"file:///sample/project/main.dart\":[1,11,14,2,18,16,3,23,17,4,40,16,6,46,16,10,69,9,11,70,9,12,71,18,13,41,19,14,42,20,95,51,58],\"file:///sample/project/clock.dart\":[96,33,12,97,52,12,100,53,16,102,54,14,104,55,17,105,34,15,106,35,16]}}");
+                                               "{\"startTime\":1000,\"events\":[1,1,2,1,3,1,4,1,6,1,10,4,11,4,12,4,13,1,14,1,95,1,96,1,97,6,100,6,102,6,104,6,105,1,106,1],\"newLocations\":{\"/sample/project/main.dart\":[1,11,14,2,18,16,3,23,17,4,40,16,6,46,16,10,69,9,11,70,9,12,71,18,13,41,19,14,42,20,95,51,58],\"/sample/project/clock.dart\":[96,33,12,97,52,12,100,53,16,102,54,14,104,55,17,105,34,15,106,35,16]}}");
```
Comment (jacob314) [inline testSrc/unit/io/flutter/perf/FlutterWidgetPerfTest.java]: The flutter sdk updated to just use paths. Updating the fake data to match the current DartVM version.

File: src/io/flutter/bazel/Workspace.java (hunk 2 above)
Comment (devoncarew) [reply src/io/flutter/bazel/Workspace.java]: Perhaps add a todo here that we anticipate a future change to this url scheme?

File: src/io/flutter/sdk/FlutterSettingsConfigurable.java [hunk 5]
```diff
@@ -230,17 +230,19 @@ public boolean isModified() {
 
   @Override
   public void apply() throws ConfigurationException {
-    final String errorMessage = FlutterSdkUtil.getErrorMessageIfWrongSdkRootPath(getSdkPathText());
-    if (errorMessage != null) {
-      throw new ConfigurationException(errorMessage);
-    }
+    if (!workspaceCache.isBazel()) {
```
Comment (devoncarew) [inline src/io/flutter/sdk/FlutterSettingsConfigurable.java]: Is this change part of another modifying the settings page?

Comment (jacob314) [reply src/io/flutter/sdk/FlutterSettingsConfigurable.java]: This was a bug fix for my previous cl. Added a comment clarifying what it is doing.

**Pull request merged**